vertical-pod-autoscaler/pkg/utils/test/test_vpa.go

```diff
@@ -38,6 +38,7 @@ type VerticalPodAutoscalerBuilder interface {
 	WithLowerBound(cpu, memory string) VerticalPodAutoscalerBuilder
 	WithTargetRef(targetRef *autoscaling.CrossVersionObjectReference) VerticalPodAutoscalerBuilder
 	WithUpperBound(cpu, memory string) VerticalPodAutoscalerBuilder
+	WithAnnotations(map[string]string) VerticalPodAutoscalerBuilder
```

Comment:
Unless I missed something, this change looks unrelated and unused.

Reply:
This is due to the change in cluster_test addVpa function - I started using builder and that function sets annotations as well.